src/openforms/forms/tests/e2e_tests/vcr_cassettes/FormDesignerRegistrationBackendConfigTests/test_configuration_objects_api_group.yaml

```diff
@@ -7,39 +7,39 @@ interactions:
       Accept-Encoding:
       - gzip, deflate, br
       Authorization:
-      - Token 171be5abaf41e7856b423ad513df1ef8f867ff48
+      - Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJ0ZXN0X2NsaWVudF9pZCIsImlhdCI6MTc2NjE0OTIwMywiZXhwIjoxNzY2MTkyNDAzLCJjbGllbnRfaWQiOiJ0ZXN0X2NsaWVudF9pZCIsInVzZXJfaWQiOiIiLCJ1c2VyX3JlcHJlc2VudGF0aW9uIjoiIn0.ax-iARZFcu5prK3nwtvVlqVnZ2svukNd2XUJ17CQOS8
       Connection:
       - keep-alive
       User-Agent:
       - python-requests/2.32.4
     method: GET
-    uri: http://localhost:8001/api/v2/objecttypes
+    uri: http://localhost:8003/catalogi/api/v1/catalogussen
```

Comment:
I am missing something here, when and why was this changed (and apparently the Authorization type)?